provider/README.md

```diff
@@ -260,7 +260,8 @@ Unless stated otherwise by the municipality, this endpoint must return only thos
 | `event_types` | Enum[] | Required | [Vehicle event(s)][vehicle-events] for state change, allowable values determined by `vehicle_state` |
 | `event_time` | [timestamp][ts] | Required | Date/time that event occurred at. See [Event Times][event-times] |
 | `publication_time` | [timestamp][ts] | Optional | Date/time that event became available through the status changes endpoint |
-| `event_location` | GeoJSON [Point Feature][geo] | Required | See also [Stop-based Geographic Data][stop-based-geo] |
+| `event_location` | GeoJSON [Point Feature][geo] | Optional | See also [Stop-based Geographic Data][stop-based-geo] |
+| `event_geographies` | UUID[] | Required | Array of Geography UUIDs consisting of every Geography that contains the location of the status change. Required if `event_location` is not present. |
```

Comment:
For symmetry I would propose that we modify `trips` to _either_ send the route (FeatureCollection with lat/lon) _or_ an array of the Properties fields.  The Properties currently contains timestamp (and optionally stop_id) so you'd also add `list-of-geographies` as the "telemetry" points, dropping `geometry`.  My read of the GeoJSON spec is that Features without `geometry` are not supported, so this would be a not-strictly-GeoJSON object.  

Example:

```js
"route": [
    {
      "timestamp": 1529968782421,
      // Required for Trips starting at a Stop
      "stop_id": "95084833-6a3f-4770-9919-de1ab4b8989b",
      "geography_ids": [
        "4ad32463-2c14-4979-9e7f-ec0f21a643e3",
        "99fc6c35-394c-4651-bfb7-9883d3c96d4a"
      ]
    },
    {
      "timestamp": 1531007628377,
      // Required for Trips ending at a Stop
      "stop_id": "b813cde2-a41c-4ae3-b409-72ff221e003d",
      "geography_ids": ["439ec866-ad7b-468f-969b-6d7e912ee5a5"]
    }
  ]
```

Reply:
I agree that there is a symmetric change to make in the Trips field. We started fleshing one out, but we decided not to include it in this PR for two reasons
- The use cases around the trips endpoint are very different
- There are some subtleties in the changes required, and these depend on the use cases. For example, one possible use case might be assessing speed limit data. It's possible to imagine adding a max speed to the properties list for this purpose, but this seems very specific and in want of some kind of generalization where the required data was specified at the Policy level. 

What do you think about starting a new issue and having a conversation around the use cases for this change?

Reply:
I'm not sure how to separate them.  Do you want to put this on the agenda for the WG meeting this coming Thursday?  Or maybe a quickie zoom session beforehand?

I appreciate wanting to keep the PR surface area low.

Reply:
Good call, def think it would be easier to flesh this out on Zoom. I will follow up via email.

Reply:
With the decision to create new event types, the work to add geographies data to the trips endpoint be captured via a future issue, if it is needed.